src/emulator/rom.c

```diff
@@ -1,433 +1,1008 @@
-#include "ram.h"
-#include "rom.h"
-#include "system.h"
-#include "xlFile.h"
-#include "xlHeap.h"
+#include "emulator/rom.h"
+#include "emulator/cpu.h"
+#include "emulator/frame.h"
+#include "emulator/ram.h"
+#include "emulator/system.h"
+#include "emulator/vc64_RVL.h"
+#include "emulator/xlCoreRVL.h"
+#include "emulator/xlHeap.h"
+#include "macros.h"
+#include "revolution/os.h"
+
+//! TODO: document these
+void fn_80063D78(s32);
+bool fn_8005F5F4(void* arg0, void* arg1, s32 arg2, void* arg3);
```

Comment:
since we have all emulator file splits, could we declare these in the right header? It seems like fn_80063D78 is in rsp.o and fn_8005F5F4 is in helpRVL.o

Reply:
I put it there because it was convenient when I needed to make an update on those prototypes, I'll move them in the proper headers (and make one for helpRVL)